actions/store_global_data_MOD.js

```diff
@@ -8,11 +8,9 @@ module.exports = {
     authorUrl: 'https://github.com/dbm-network/mods',
     downloadURL: 'https://github.com/dbm-network/mods/blob/master/actions/store_global_data_MOD.js',
   },
-
   subtitle(data, presets) {
     return presets.getVariableText(data.storage, data.varName);
   },
-
   variableStorage(data, varType) {
```

Comment:
```suggestion

  variableStorage(data, varType) {
```

Reply:
I think this is all the changes I'd like, i'll take a look after these are added and then approve the PR.